copy.js

```diff
@@ -0,0 +1,4 @@
+(function() {
```

Comment:
TBH I wouldn't use a separate file for this, to minimize downloads. Just use the existing `app.js`